ci/multibuild_config.sh

```diff
@@ -0,0 +1,22 @@
+# Define custom utilities
+# Test for OSX with [ -n "$IS_OSX" ]
+
+function pre_build {
+    # Any stuff that you need to do before you start building the wheels
+    # Runs in the root directory of this repository.
+    if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$PLAT" == "i686" ]; then
+        apt install gcc-multilib g++-multilib
```

Comment:
@madig why do you need to do `apt install gcc-multilib g++-multilib` inside the manylinux1 docker images? Firstly, these are CentOS based and thus they use `yum`, not `apt`. But also, multibuild already should pull the i686 version of the manylinux1 dockerfile automatically, which should have all the 32 bit toolchain already set up...

Reply:
Hm. The test below also doesn't make much sense as it's also run in manylinux1. The load test still passes so I guess it's using the installed FT.